src/test/java/jimmy/mcgymmy/model/UserPrefsTest.java

```diff
@@ -19,4 +21,13 @@ public void setMcGymmyFilePath_nullPath_throwsNullPointerException() {
         assertThrows(NullPointerException.class, () -> userPrefs.setMcGymmyFilePath(null));
     }
 
+    @Test
+    public void equals_returnsCorrectResult() {
+        UserPrefs userPref = new UserPrefs();
+        // identical -> returns true
+        assertEquals(userPref, userPref);
```

Comment:
I think we should add a test case where there are 2 userPrefs with the same values but not the same object.

Reply:
added!